### DISCUSSION TITLE
Theme does not support Chinese character display.

### DISCUSSION THREAD

**Discussion by ghost** [vincenzopalazzo/material-ui-swing#161] [Q&A]
<img width="441" alt="Screen Shot 2021-04-15 at 15 15 52" src="https://user-images.githubusercontent.com/25057326/114829348-8777e380-9dfd-11eb-99eb-318fd4deb441.png">

**Comment by vincenzopalazzo** [comment]
Hello @echo-cool,

The motivation for the font error is that the library uses a custom font under the hood.

But the library has a cool method that managed the customization, which is described in this [old issue](https://github.com/vincenzopalazzo/material-ui-swing/issues/87#issuecomment-549139460).

You can choose the font that you want, but I suggest you to use a custom font that is supported from all the platform, or at list one of your platform support.

Another example is inside the  [DarkStackOverflowTheme](https://github.com/material-ui-swing/DarkStackOverflowTheme/blob/430de04c77f1013faed66e74a4091734a01568d4/src/main/java/io/materialtheme/darkstackoverflow/DarkStackOverflowTheme.java#L41-L58) that use another custom font that is in a local directory like [this](https://github.com/material-ui-swing/DarkStackOverflowTheme/tree/master/src/main/resources/font)

So, I think that the solution at your question is a clean solution offers from the library.

@echo-cool I answer your question? If you have other question, please let me know :)

**Reply by ghost** [reply]
Well, It perfectly solved that problem,
Many thanks !
